.rubocop.yml

```diff
@@ -21,6 +21,7 @@ AllCops:
     - 'tool/**/*'
     - 'truffleruby/target/**/*'
     - 'truffleruby-gem-test-pack-*/**/*'
+    - 'work/**/*'
```

Comment:
What's this for?

Reply:
Like 2f04d51 says. I plan to switch to an include list within jt so this is no longer a problem, but that's for another PR.

Reply:
Sorry I still don't really get it. You have some extra files locally and you don't want to check them with RuboCop? This seems like something super-specific to you. Can't you keep this files outside the repo?

Reply:
It's really not convenient to keep them outside the repo. It's similar to the top rule, `*.rb`.
But I'll just remove this commit then.